<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Likes</class>
 <widget class="QWidget" name="Likes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>407</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 102, 102);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic10.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic11.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic3.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic5.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic4.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Developer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Hamaid Rezaei</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Programmer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Master</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Driver</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Developer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Hesam Mohammadi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Sam saberi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>pouria gholami</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Ali zanganeh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Donya alizadeh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Hesam koni</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/pic7.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Doctor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Mahdis Moradi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/photo/6d639d40-4fa2-11ee-8a5b-cd5f522ee0ba.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Designer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>label_13</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>label_8</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>label_14</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>label_9</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>label_15</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>label_10</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>label_16</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>label_11</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>label_17</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>label_12</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>label_18</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>label_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>label_19</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>label_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>label_20</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
